_posts/2024-08-06-release.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Release 2024.32
+author: Priya
+date: 2024-08-06
+category: Jekyll
+layout: post
+---
+### Enhancements:
+* Data usage now tracks row counts instead of data size, with restrictions applied at the workspace level to stop Mammoth usage when limits are exceeded.
```

Comment:
"Changed the primary pricing criteria from storage to number of rows, to encourage transparency and relatability in pricing"